pandas/tseries/index.py

```diff
@@ -64,6 +64,7 @@ def f(self):
             if self.tz is not utc:
                 values = self._local_timestamps()
 
+        # boolean accessors -> return array
```

Comment:
ahh ok. pls open a new issue and I will do a followup to fixup this. It much too specially casey. So good to go when you are ready.